src/components/Sample.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export const Sample = (props: { onClick: () => void; btnText: string }) => {
+  return (
+    <button type="button" onClick={props.onClick}>
+      {props.btnText}
+    </button>
+  );
+};
```

Comment:
Just a sample component so we can also set some unit tests to make sure that the testing framework is correctly set and also as an excuse to add a few sample tests that work as documentation at the same time.